cinema_service/urls.py

```diff
@@ -5,4 +5,6 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
The `include` function with a `namespace` argument requires the included URLconf to define an `app_name` attribute. Ensure that `cinema.urls` has an `app_name` defined, or remove the `namespace` argument if not needed.